Configurations/unix-Makefile.tmpl

```diff
@@ -616,32 +610,25 @@ uninstall_docs: uninstall_man_docs uninstall_html_docs
 
 {- output_off() if $disabled{fips}; "" -}
 install_fips: build_sw $(INSTALL_FIPSMODULECONF)
+	@$(ECHO) "*** Installing FIPS module and its configuration ($@)"
```

Comment:
`$@` doesn't look right, there...

Reply:
The `$@` shows what target is being built.

Reply:
Yeah, right after `its configuration`...  I'd expect the parenthesis to have the file name of the configuration with that sentence